python-frontend/src/main/java/org/sonar/python/tree/StringElementImpl.java

```diff
@@ -65,7 +75,8 @@ public List<Tree> computeChildren() {
     // Warning: in the case of f-strings, there's a kind of overlap between `token` and `formattedExpressions`: they
     // are different representations of the same analyzed code.
     // TreeUtils.tokens() doesn't contain the tokens of the formattedExpressions.
-    return Stream.concat(Stream.of(token), formattedExpressions.stream()).collect(Collectors.toList());
+    return Stream.concat(Stream.of(token), Stream.concat(fStringMiddles.stream(), Stream.of(fstringEnd)))
```

Comment:
Ok so this does not work beacause of the creation of list with null values. It will not work either with Stream of Streams because the order will not be kept, so I went the old school way :)